### PR TITLE
Update test dependencies

### PR DESCRIPTION
**Pull request by Richienb**

### CODE REVIEW

**Comment by Qix-** [comment]
I don't agree with a number of these changes, personally - namely some of the changes in import semantics and the syntactic changes of arrow functions. I'll let @sindresorhus decide though, because they may be related to Ava and/or XO, both of which he maintains.

**Comment by Richienb** [comment]
@Qix- The import statements become require statements because ava 3.x no longer supports esm and the syntactic changes of arrow functions are enforced by the newest version of xo.

**Comment by Qix-** [comment]
Interesting, I don't use Ava myself so I wasn't aware. Even more reason for @sindresorhus to review, then :)

**Comment by sindresorhus** [comment]
I'm holding off on upgrading AVA until next April when I can just convert everything to native ESM. If we convert all tests to require now, it will just be a lot of churn when we convert back again in half a year. I'll take the other changes though.

File: package.json [hunk 1]
```diff
@@ -62,7 +62,8 @@
 			"@typescript-eslint/member-ordering": "off",
 			"no-redeclare": "off",
 			"unicorn/string-content": "off",
-			"unicorn/better-regex": "off"
+			"unicorn/better-regex": "off",
+			"unicorn/prefer-number-properties": "off"
```
Comment (sindresorhus) [inline package.json:66]: ?

Comment (Richienb) [reply package.json:66]: In `templates.js`, can we replace `parseInt` with `Number.parseInt`?

Comment (sindresorhus) [reply package.json:66]: Yes

Comment (Richienb) [reply package.json:66]: ok

**Comment by Qix-** [comment]
Thanks @Richienb :)

**Pull request merged**